lib/v2/liquipedia/cs_matches.js

```diff
@@ -0,0 +1,54 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const team = ctx.params.team ?? 'G2_Esports';
+
+    const rootUrl = 'https://liquipedia.net';
+    const currentUrl = `${rootUrl}/counterstrike/${team}/Matches`;
+
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+    });
+
+    const $ = cheerio.load(response.data);
+    const list = $('.recent-matches-bg-lose, .recent-matches-bg-win');
+
+    const matches = list
+        .map((_, item) => {
+            item = $(item);
+
+            const getRes = () => {
+                if (item.attr('class') === 'recent-matches-bg-lose') {
+                    return 'LOSS';
+                } else if (item.attr('class') === 'recent-matches-bg-win') {
+                    return 'WIN';
+                } else {
+                    return 'DRAW';
```

Comment:
Unreachable condition since `.recent-matches-bg-tie` is not selected in https://github.com/DIYgod/RSSHub/blob/068457cd277dc76efe33e3658f54f59970ac9895/lib/v2/liquipedia/cs_matches.js#L16

Reply:
Updated.